documentation/source/NuRadioMC/pages/HDF5_structure.rst

```diff
@@ -72,14 +72,26 @@ The top-level attributes can be accessed using ``f.attrs``. These contain:
             ``start_event_id`` | ``event_id`` of the first event in the file
             ``trigger_names`` | List of the names of the different triggers simulated
             ``Tnoise`` | (explicit) noise temperature used in simulation
-            ``Vrms`` |  RMS of the voltage used as thermal noise floor. Determine from ``Tnoise`` and ``bandwidth``
-            ``bandwidth`` | Bandwidth of the antennas/detector (for triggering)
             ``n_samples`` | Samples of the to-be generated antenna signals
             ``config`` | The (yaml-style) config file used for the simulation
             ``deposited`` |
             ``detector`` | The (json-format) detector description used for the simulation
             ``dt`` | The time resolution, i.e. the inverse of the sampling rate used for the simulation. This is not necessarily the same as the sampling rate of the simulated channels!
 
+
+The station-level attributes can be accessed using ``f[station_<station_id>].attrs``. The first two attributes ``Vrms`` and ``bandwidth`` also exist on the top-level and refer to the corresponding to the first station/channel pair.
+
+    .. _hdf5-station-attrs-table:
+
+    .. csv-table:: HDF5 station attributes
+            :header: "Key", "Description"
+            :widths: auto
+            :delim: |
+
+            ``Vrms`` | RMS of the voltage used as thermal noise floor :math:`v_{n} = (k_{B} \, R \, T \, \Delta f) ^ {0.5}`. See the relevant section "Noise voltage and power" in this `wiki article <https://en.wikipedia.org/wiki/Johnson%E2%80%93Nyquist_noise>`_ (last two equations). Determine from ``Tnoise`` and ``bandwidth`` (see below).
```

Comment:
I missed that you already updated the documentation, sorry about that. Looks good.